ext/odbc/php_odbc.c

```diff
@@ -2972,59 +3000,63 @@ PHP_FUNCTION(odbc_foreignkeys)
 
 	if (result->numcols > 0) {
 		if (!odbc_bindcols(result)) {
-			efree(result);
+			zval_ptr_dtor(return_value);
 			RETURN_FALSE;
 		}
 	} else {
 		result->values = NULL;
 	}
 	result->conn_ptr = conn;
 	result->fetched = 0;
-	RETURN_RES(zend_register_resource(result, le_result));
 }
 /* }}} */
 #endif /* HAVE_SOLID */
 
 /* {{{ Returns a result identifier containing information about data types supported by the data source */
 PHP_FUNCTION(odbc_gettypeinfo)
 {
+	zend_value_error("Nope");
+	RETURN_THROWS();
+
```

Comment:
Debug code?

Reply:
shit.. yes it is. I had a mysterious ZPP mismatch error and I desperately tried to find it.. I'm not sure anymore where the issue was exactly, but it was somewhere deeper than this :)